spec/system/user_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'user pages' do
+  let(:membership) { create :membership, admin: true }
+  let!(:admin) { create :user, first_name: 'tester', memberships: [membership] }
```

Comment:
This person does _not_ need a first name of "tester." That was a leftover from when I was trying different ideas for setting the current user.

Reply:
ok